docs/data/date-pickers/date-time-picker/CustomDateTimePicker.tsx

```diff
@@ -18,28 +18,34 @@ export default function CustomDateTimePicker() {
   return (
     <LocalizationProvider dateAdapter={AdapterDateFns}>
       <Stack spacing={3}>
-        <DateTimePicker
-          disableFuture
-          hideTabs
-          openTo="hours"
-          value={value}
-          onChange={(newValue) => {
-            setValue(newValue);
+        <LocalizationProvider
+          dateAdapter={AdapterDateFns}
+          localeText={{
+            openPreviousView: 'Open previous view',
```

Comment:
This one was weird, we were setting a label related to months on a picker where we can never access a day view.

I don't even now why we do this example here, it should be using `TimePicker`.
But I don't want to do deeper changes on a l10n pull request.

Reply:
Why not just remove it? Those texts are the default ones. Modifying `leftArrowButtonText` in this doc example was helpful because the component was responsible for translations. 
Now we have a dedicated page to document how text are modified

Reply:
Indeed
Removed